### PR TITLE
Accra integration tests using unittests

### PR DESCRIPTION
**Pull request by prhod**
This PR aim to check Accra generated Data's consistency with osm2gtfs evolutions.
The same tests framework as https://github.com/grote/osm2gtfs/pull/78  is used.
Those tests will ease validations of modifications using Overpass Mocks and an expected GTFS result generated with the provided mocks. 
There are some code duplications, but avoiding them would conflicts with in progress PR ;-)

### CODE REVIEW

File: osm2gtfs/tests/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+Accra
+------
+#### Description
+
+There are 3 unittests for the Accra GTFS generation
+
+1. Generation of the cache stops Data from an Overpass result Mock
+* Generation of the cache routes Data from an Overpass result Mock
+* Generation of the GTFS ZIP File using the previously generated cache data
```
Comment (grote) [inline osm2gtfs/tests/README.md]: The enumeration does seem to be broken here.

**Comment by xamanu** [comment]
Very nice!

**Comment by grote** [comment]
@prhod how about adding these to the CI?

**Comment by prhod** [comment]
Thanks for the merge. I'll create an other PR with the fix of the readme and the addition of the tests in the CI. I'm glad you suggested it, I was'nt confident that it was a good idea ;)

File: osm2gtfs/tests/README.md (hunk 1 above)
Comment (prhod) [reply osm2gtfs/tests/README.md]: sorry for that, my markdown editor was ok with this syntaxe

**Comment by grote** [comment]
> I'm glad you suggested it, I was'nt confident that it was a good idea ;)

No, I think it is a good idea. If a new PR breaks Accra, we want to know right away without doing manual tests.

You can create another test stage for unit tests.

**Comment by xamanu** [comment]
Yes, please add to the CI, this gives immediate feedback. Thanks!

**Comment by prhod** [comment]
Readme fix and CI integration available in https://github.com/grote/osm2gtfs/pull/93 :)

**Pull request merged**